web_external/Viewer/index.jsx

```diff
@@ -63,9 +64,8 @@ class Viewer extends PureComponent {
                             }}
                             annotationsSelect={this.props.annotationsSelect}
                             key={this.props.itemModel.id} />,
-                        this.state.ready && !this.props.annotationGeometryContainer
-                        && <div className='no-annotation-message' key='no-annotation-message'>
-                            <span>No annotation</span>
+                        message && <div className='message' key='message'>
```

Comment:
Do we really want the 'loading' message to be red also, or should we have different classes for different types of message? (If you like, I could do that in a follow-up PR...)